user_util/user_util.py

```diff
@@ -1,3 +1,33 @@
 # -*- coding: utf-8 -*-
 
 """Main module."""
+import hashlib
+
+
+RETIRED_USERNAME_FMT = 'retired_username_{}'
+RETIRED_EMAIL_FMT = 'retired_email_{}@edx.org'
```

Comment:
It would be nice if we could make these settings somehow. If nothing else I'd change the email to something like "...@blackhole.edx.org" so it can be easily filtered.

Reply:
Agreed! Will do.

Reply:
Did - used `retired.edx.org` to keep its utility explicit.